### PR TITLE
[HybridConnectivity] Update codeowner

### PR DESCRIPTION
**Pull request by alexmahonic**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1073,10 +1073,10 @@
 # ServiceOwners:                                                   @sahilarora92 @dhruvil009 @rahuls-microsoft
 
 # PRLabel: %HybridConnectivity
-/sdk/hybridconnectivity/Azure.ResourceManager.*/                   @alexmahonic
+/sdk/hybridconnectivity/Azure.ResourceManager.*/                   @alexmahonic @PriyankaHariharan @apmehrotra
```
Comment (jsquire) [inline .github/CODEOWNERS:1076]: - `apmehrotra` does not have the public necessary group memberships nor write access and is ineligible to be an owner.  

- `PriyankaHariharan` does not have a public membership to the Microsoft organization, which is required.

Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.
- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

You can verify the state of the account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.

**Review by jsquire — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (PriyankaHariharan) [reply .github/CODEOWNERS:1076]: Updated my GH `PriyankaHariharan` to be public for Microsoft organization!

**Review by jsquire — APPROVED** [review]
Account issues have been resolved.

File: .github/CODEOWNERS (hunk 1 above)
Comment (jsquire) [reply .github/CODEOWNERS:1076]: Accounts all look to be in good standing now.

**Pull request merged**